helix-front/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Here I recommend we accept the default options https://prettier.io/docs/en/options.html. This can be configured in the future if necessary.